### PR TITLE
feat: add options for hot keys

### PR DESCRIPTION
**Pull request by iammapping**
Allow set custom hot keys.

The default behaviors are same with previous: Ctrl(⌘)+K or s,/ when not editing, but add abilities to disable the hot keys or set custom ones now

### CODE REVIEW

File: src/DocSearchButton.tsx [hunk 1]
```diff
@@ -46,10 +48,12 @@ export const DocSearchButton: Component<DocSearchButtonProps> = ({
         <MagnifierIcon class="docsearch-modal-btn-icon" />
       </span>
       <span class="docsearch-btn-placeholder"> {buttonText} </span>
-      <span class="docsearch-btn-keys">
-        <kbd class="docsearch-btn-key">{ctrlKey()}</kbd>
-        <kbd class="docsearch-btn-key">K</kbd>
-      </span>
+      {ctrlWithKey && (
```
Comment (amrbashir) [inline src/DocSearchButton.tsx]: Can we add a fallback here? if `ctrlWithKey` is disabled, we should show one of the singleKeys, probably the first one.

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you for your contributions. Could you also add a change file in `.changes` directory?

File: src/DocSearchButton.tsx (hunk 1 above)
Comment (iammapping) [reply src/DocSearchButton.tsx]: Good idea, this will make the hot keys clearer. Wait a minute, let me modify it

File: README.md [hunk 2]
```diff
@@ -41,6 +41,11 @@ docsearch({
   host: "YOUR_HOST_URL",
   apiKey: "YOUR_SEARCH_API_KEY",
   indexUid: "YOUR_INDEX_UID",
+  hotKeys: {
+    // set false to disable
+    ctrlWithKey: "k",
+    singleKeys: ["s", "/"],
+  }
```
Comment (amrbashir) [inline README.md]: Let's remove this, I think we should keep the examples as concise and the defaults are good enough for most usecases.

**Review by amrbashir — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (amrbashir) [inline README.md]:
```suggestion
```

**Review by amrbashir — APPROVED** [review]
Thank you

**Pull request merged**